modules/virt-viewing-network-state-of-node-console.adoc

```diff
@@ -15,5 +15,23 @@ In the *NodeNetworkState* page, you can view the list of `NodeNetworkState` reso
 
 . To access the detailed information about `NodeNetworkState` resource, click the `NodeNetworkState` resource name listed in the *Name* column .
 
-. to expand and view the *Network Details* section for the `NodeNetworkState` resource, click the *>* icon . Alternatively, you can click on each interface type under the *Network interface* column to view the network details.
+. To expand and view the *Network Details* section for the `NodeNetworkState` resource, click the greater than (*>*) symbol . Alternatively, you can click on each interface type under the *Network interface* column to view the network details.
+
+== Viewing a graphical representation of the NNS topology
```

Comment:
@sbeskin-redhat Can you slap an [anchor ID on this heading](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings) before we merge?